stable/mysql/values.yaml

```diff
@@ -20,12 +20,14 @@ testFramework:
 ##
 ## Default: random 10 character string
 # mysqlRootPassword: testing
+allowEmptyRootPassword: false
```

Comment:
So again, I don't think we need new values. Using `mysqlAllowEmptyPassword` should be fine